### PR TITLE
Update etc/state-migration-test/Cargo.lock

### PR DESCRIPTION
**Pull request by birchmd**
Looks like this got missed in https://github.com/aurora-is-near/aurora-engine/pull/183

### CODE REVIEW

File: etc/state-migration-test/Cargo.lock [hunk 1]
```diff
@@ -708,7 +708,7 @@ dependencies = [
 [[package]]
 name = "rjson"
 version = "0.3.2"
-source = "git+https://github.com/aurora-is-near/rjson#cc3da9495e7e520900d66d2b517539f74fff93cf"
+source = "git+https://github.com/aurora-is-near/rjson?rev=cc3da949#cc3da9495e7e520900d66d2b517539f74fff93cf"
```
Comment (sept-en) [inline etc/state-migration-test/Cargo.lock:711]: Shouldn't it be something like

```suggestion
source = "git+https://github.com/aurora-is-near/rjson?rev=cc3da9495e7e520900d66d2b517539f74fff93cf"
```

Comment (birchmd) [reply etc/state-migration-test/Cargo.lock:711]: The lock file is automatically generated so I can only assume it's right as is

Comment (sept-en) [reply etc/state-migration-test/Cargo.lock:711]: Ok, it didn't generate this change for me, but I'm ok with it :)

**Review by sept-en — APPROVED** [review]

**Comment by joshuajbouw** [comment]
I don't think that this is quite required anymore as the `Cargo.lock` will just be naturally updated.

**Pull request closed without merging**